web-api/src/business/useCases/caseAdvancedSearchInteractor.ts

```diff
@@ -80,5 +90,16 @@ export const caseAdvancedSearchInteractor = async (
     MAX_SEARCH_RESULTS,
   );
 
-  return filteredCases;
+  return filteredCases.map(filteredCase => {
+    return {
+      caseCaption: filteredCase.caseCaption,
+      docketNumber: filteredCase.docketNumber,
+      docketNumberWithSuffix: filteredCase.docketNumberWithSuffix,
+      petitionerNames: filteredCase.petitioners?.map(p => p.name),
+      petitionerStateNames: filteredCase.petitioners?.map(
+        p => US_STATES[p.state] || p.state,
+      ),
+      receivedAt: filteredCase.receivedAt,
+    };
+  });
```

Comment:
Total personal preference and absolutely not necessary to change, but I like using shorthand where possible.

This:
```typescript
return someArray.map(m => {
  return {
    foo: 'bar'
  };
});
```

can be shortened by wrapping the return object in parens:

```typescript
return someArray.map(m => ({
  foo: 'bar'
}));
```